window/src/os/windows/wgl.rs

```diff
@@ -277,10 +295,11 @@ impl GlState {
         }
 
         let res = unsafe { SetPixelFormat(hdc, format_id, &pfd) };
-        if res == 0 {
+        if res.is_err() {
```

Comment:
For something like this, where we are now getting an error as a result, I would prefer something like:

```rust
if let Err(err) = res {
   anyhow::bail!("SetPixelFormat function failed: {err:#}");
}
```

Reply:
or perhaps even just:

```rust
unsafe { SetPixelFormat(hdc, format_id, &pfd) }.context("SetPixelFormat")?;
```

(and can we remove the `unsafe` from here?)